### PR TITLE
Handle wallet disconnect in withdrawal flow

### PR DESCRIPTION
**Pull request by jurgenwerk**
Ticket [CS-1411](https://linear.app/cardstack/issue/CS-1411/when-workflow-epilogue-shows-includes-wallet-card-as-a-summary-card).

This is my first PR, be gentle 😂 🙆‍♂️

This PR addresses an edge case bug in card issuance and withdrawal flow. The problem appears when a user completes the flow, and then disconnects the layer 2 wallet in their Card app: the "Connect Card Wallet" prompt appears, which doesn't make much sense when the user already completed the flow, and causes confusion.

**Current behaviour:**

In the issuance or withdrawal flow, the wallet summary appears at the end. 

image

If the user disconnects their wallet (in the mobile app or using the Disconnect wallet), the connect wallet prompt appears: 

image

**After the fix:**

My proposed fix is that when the workflow is completed and the user disconnects their wallet, a card summary appears, but without the address and balance (because this data gets wiped out on disconnect). The status now says "Disconnected" (there's a new red icon for this):

image

⁉️ **Questions**:

1. Who does usually need to review and approve small design adjustments like this one? @Aierie suggested I ask in the design channel, but is it ok if we discuss here?
2. Perhaps the layer1 connect suffers from the same problem (I haven't checked yet). Does it make sense to do the same there?

### CODE REVIEW

File: packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs [hunk 1]
```diff
@@ -47,8 +47,8 @@
         {{!-- future: show Prepaid Cards, Badges --}}
         <CardPay::LabeledValue
           @label="Status"
-          @value="Connected"
-          @icon="success-bordered"
+          @value={{if this.isConnected "Connected" "Disconnected"}}
+          @icon={{if this.isConnected "success-bordered" "failure-bordered"}}
```
Comment (jurgenwerk) [inline packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs:52]: While the red icon could easily be just a couple of lines of CSS, I added a new SVG icon for the sake of consistency. What are your thoughts on this?

Comment (Aierie) [reply packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs:52]: New SVG icon sounds better than constructing a circle with CSS! CSS circles can be a bit fragile.

**Comment by jurgenwerk** [comment]
@lukemelia this is a different approach from what we discussed earlier (i.e. creating a new summary card component and including it in the epilogue of the workflows), but after some research I realized there's a chance I'd fall in a refactoring rabbit hole and spend a considerate amount of time trying to split up the `CardPayLayerTwoConnectCardComponent` 🤔 so I think this change is simple enough we can keep it in the original component, but let me know if you think otherwise!

**Comment by Aierie** [comment]
> 2. Perhaps the layer1 connect suffers from the same problem (I haven't checked yet). Does it make sense to do the same there?

Yep, layer 1 should have the same problem in the deposit flow. A similar solution might be good!

**Review by lukemelia — APPROVED** [review]
Well-explained PR! I appreciate the instinct to make the smallest amount of changes to satisfy the requirements. I tried it out and it behaves well in all the scenarios I tried. Once we address the svg changes under discussion, let's merge and do the same thing for layer 1. 👍

**Comment by lukemelia** [comment]
> Who does usually need to review and approve small design adjustments like this one? @Aierie suggested I ask in the design channel, but is it ok if we discuss here?

Our designer doesn't typically review PRs, so it would be good to share a screenshot in the design-internal channel and ask for feedback on this new state.

**Review by burieberry — APPROVED** [review]
Looks good! We can merge if the svg is updated. I also like the minimal amount of changes, and the red circle idea for disconnection.

You can tag me to review any design related PRs or for any questions you might have. The design channel, as others mentioned, is also good for asking questions.

**Review by Aierie — APPROVED** [review]
Sorry, realised I forgot to add an approval after leaving comments. Other than the svg viewBox, this looks good to me!

**Comment by jurgenwerk** [comment]
@Aierie @lukemelia @burieberry thanks! Will wait for the designer's opinion and then fix the SVG.

File: packages/web-client/public/images/icons/failure-bordered.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20"><g fill="#ff4040" stroke="#ffa7a7"><circle cx="10" cy="10" r="10" stroke="none"/><circle cx="10" cy="10" r="9.5" fill="none"/></g></svg>
```
[the remark this answers is not in the document]
Comment (jurgenwerk) [reply packages/web-client/public/images/icons/failure-bordered.svg:1]: Nice catch! Will fix.

[the remark this answers is not in the document]
Comment (jurgenwerk) [reply packages/web-client/public/images/icons/failure-bordered.svg]: Added.

**Comment by jurgenwerk** [comment]
I replaced the icon with the one from the project's set of icons for system messages:

image

**Pull request merged**